kolejka/worker/stage0.py

```diff
@@ -122,6 +127,12 @@ def stage0(task_path, result_path, temp_path=None, consume_task_folder=False):
         docker_call += [ '--init' ]
         if task.limits.cpus is not None:
             docker_call += [ '--cpuset-cpus', ','.join([str(c) for c in cgs.limited_cpuset(cgs.full_cpuset(), task.limits.cpus, task.limits.cpus_offset)]) ]
+
+        if task.limits.gpus is not None:
+            check_gpu_runtime_availability()
+            gpus = ','.join(gpu.limited_gpuset(gpu.full_gpuset(), task.limits.gpus, task.limits.gpus_offset))
+            docker_call += [ '--runtime=nvidia', '--shm-size=1g', '--gpus', f'\'"device={gpus}"\'' ]
```

Comment:
To note about `--shm-size=1g` `pytorch` need to be set up at least 1g

Reply:
@grzegorz-gutowski It requires additional work to handle shared memory limits. The reason it was added: https://pytorch.org/elastic/0.2.0rc1/examples.html
> PyTorch data loaders use shm. The default docker shm-size is not large enough and will OOM when using multiple data loader workers. You must pass --shm-size to the docker run command or set the number of data loader workers to 0 (run on the same process) by passing the appropriate option to the script (use the --help flag to see all script options). In the examples below we set --shm-size.